mapdata/WarcraftLegacies/AbilityData/Skin/843863129.json

```diff
@@ -29,7 +29,7 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "The silithid spawns 3 Swarm Beetles upon death. They last for 35 seconds."
+      "Value": "The silithid spawns 3 Swarm Beetles upon death. They last for 15 seconds."
```

Comment:
You don't have to fix it but... data fields!!!!!! This is not a tooltip we should have to update manually